src/PscIde/Command.purs

```diff
@@ -348,14 +365,27 @@ instance decodeRebuildError :: DecodeJson RebuildError where
     o <- decodeJson json
     message <- o .? "message"
     errorCode <- o .? "errorCode"
+    errorLink <- o .? "errorLink"
     moduleName <- o .? "moduleName"
     filename <- o .? "filename"
     position <- pure $ eitherToMaybe do
       p <- o .? "position"
-      { line: _, column: _ } <$> p .? "startLine" <*> p .? "startColumn"
-    pure (RebuildError { errorCode, moduleName, filename, message, position})
+      { startLine: _, startColumn: _, endLine: _, endColumn: _ } <$> p .? "startLine" <*> p .? "startColumn" <*> p .? "endLine" <*> p .? "endColumn"
+    pursIde <- pure $ eitherToMaybe do
+      pio <- o .? "pursIde"
+      name <- pio .? "name"
+      completions <- pio .? "completions"
+      pure $ PursIdeInfo { name, completions }
+    suggestion <- pure $ eitherToMaybe do
+      so <- o .? "suggestion"
+      replacement <- so .? "replacement"
+      rr <- so .? "replaceRange"
+      replaceRange <- pure $ eitherToMaybe $ { startLine: _, startColumn: _, endLine: _, endColumn: _ } <$> rr .? "startLine" <*> rr .? "startColumn" <*> rr .? "endLine" <*> rr .? "endColumn"
+      pure $ PscSuggestion { replacement, replaceRange }
+    pure (RebuildError { errorCode, errorLink, moduleName, filename, message, position, pursIde, suggestion })
     where
-      eitherToMaybe = either (const Nothing) Just
+    eitherToMaybe :: forall a b. Either a b -> Maybe b
```

Comment:
This is in `Data.Either` as `hush` now :)

Reply:
Ah, I knew it from purescript-errors, wasn't aware it was moved over